doc/orchestrator.rst

```diff
@@ -1,60 +1,324 @@
+can address the cluster with the SmartRedis clients like a single block of memory
+using simple put/get semantics in SmartRedis. The client establishes a
+connection using the database address and travels off the model compute node to
+the database compute node.
 
 .. |cluster-orc| image:: images/clustered-orc-diagram.png
   :width: 700
   :alt: Alternative text
 
 |cluster-orc|
 
+A clustered database is optimal for high data throughput scenarios
+such as online analysis, training and processing.
+Clustered Orchestrators support data communication across multiple simulations.
+With clustered database deployment, SmartSim can run AI models, and Torchscript
+code on the CPU(s) or GPU(s) with existing data in the ``Orchestrator``.
+Produced data can then requested by another application.
 
-With a clustered ``Orchestrator``, multiple compute hosts memory can be used together
-to store data. As well, the CPU or GPU(s) where the ``Orchestrator`` is running can
-be used to execute the AI models, and Torchscript code on data stored within it.
+-------
+Example
+-------
+We provide a demonstration on automating the deployment of
+a standard Orchestrator from within a Python driver script. Once the standard database is started,
+we demonstrate connecting a client from within the application script to transmit and poll data.
 
-Users do not need to know how the data is stored in a clustered configuration and
-can address the cluster with the SmartRedis clients like a single block of memory
-using simple put/get semantics in SmartRedis. SmartRedis will ensure that data
-is evenly distributed amongst all nodes in the cluster.
+The example is comprised of two script files:
+
+* The Application Script
+* The Experiment Driver Script
+
+**The Application Script Overview:**
+The example application script is a Python file that contains
+instructions to create and connect a SmartRedis
+client to the standard Orchestrator to transmit and retrieve
+tensors.
+
+**The Application Script Contents:**
+
+1. Connecting a SmartRedis client within the application to send and retrieve a tensor
+   from the standard database.
+
+**The Experiment Driver Script Overview:**
+The experiment driver script launches and manages
+the example entities with the ``Experiment`` API.
+In the driver script, we use the ``Experiment``
+to create and launch a ``Model`` instance
+to communicate with a launched standard ``Orchestrator``.
+
+**The Experiment Driver Script Contents:**
+
+1. Launching a ``Model`` and ``Orchestrator``.
+
+The Application Script
+======================
+To begin writing the application script, import the necessary packages:
+
+.. code-block:: python
+
+  from smartredis import Client, log_data
+  from smartredis import *
+  import numpy as np
+
+Initialize the Client
+---------------------
+To establish a connection with the standard database,
+we need to initialize a new SmartRedis client.
+Since the standard database we launch in the driver script
```

Comment:
"Since the standard database we launch in the driver script
multi-sharded," might be "Since the standard database launch*ed* in the driver script
is *not* sharded"